README.md

```diff
@@ -116,6 +116,8 @@ As a starting point, we recommend [The Business Model Canvas](https://www.strate
 
 ![The Business Model Canvas](resources/business-model-canvas.png)
 
+Available templates can be found in [this repository](https://github.com/ddd-crew/virtual-modelling-templates)
```

Comment:
We don't provide a template for the Business Model Canvas. We use the one from Strategyzr https://www.strategyzer.com/canvas. Miro also provides a BMC template as well as some other tools.

I think there are 2 choices here:

1. Add a link to Strategyzr.com

2. Don't add a link and trust people will see the link Strategyzr.com in the image itself

Reply:
👍

Reply:
```suggestion
```